keras/layers/convolutional.py

```diff
@@ -579,8 +579,8 @@ class Conv2D(Conv):
       value != 1 is incompatible with specifying any `dilation_rate` value != 1.
     padding: one of `"valid"` or `"same"` (case-insensitive).
       `"valid"` means no padding. `"same"` results in padding with zeros evenly
-      to the left/right or up/down of the input such that output has the same
-      height/width dimension as the input.
+      to the left/right or up/down of the input. The output has the same
+      height/width dimension as the input when `"same"` and `strides=(1,1)`.
```

Comment:
"When `padding="same"` and `strides=1`, the output has the same size as the input."